packages/api/endpoints/s3credentials.js

```diff
@@ -40,9 +40,12 @@ async function requestTemporaryCredentialsFromNgap(username) {
 async function s3credentials(req, res) {
   const username = req.authorizedMetadata.userName;
   const credentials = await requestTemporaryCredentialsFromNgap(username);
-  return res.send(JSON.parse(credentials.Payload));
+  const creds = JSON.parse(credentials.Payload);
+  if (Object.keys(creds).some((key) => ['errorMessage', 'errorType', 'stackTrace'].includes(key))) {
+    log.error(credentials.Payload);
```

Comment:
Just to make sure I understand: if we get here, then `credentials.Payload` is actually an error object? If so, would it make sense to include that as part of the response from the server so the user would have more helpful errors?

Reply:
We could, I think we even talked about this.  just append it to the boom below.

Reply:
b1e0db147